.tool-versions

```diff
@@ -1 +1 @@
-nodejs 16.14.2
+nodejs >=18.0.0
```

Comment:
Could you verify if this works?
I never use this before, and I don't see it in [this docs](https://asdf-vm.com/manage/configuration.html#tool-versions) as well 🙏

Reply:
Actually, I changed from asdf to nvm since I started to work with easyHotel but to verify that .tool-versions is working you can run `asdf current` this command will look for all packages in your .tool-versions (global version and local version) but since I changed to nvm should I remove .tool-versions ? 
![image](https://user-images.githubusercontent.com/122268989/217415229-722f1c87-fb22-4aed-ab36-9c69a55e4fa3.png)

Reply:
I meant you are using the wrong syntax for this. It should be:
```suggestion
nodejs 18.0.0 # or 18.14.0 (latest LTS)
```

Reply:
The version is now updated 🙏